vllm/model_executor/models/siglip.py

```diff
@@ -443,27 +443,19 @@ def __init__(
         self.config = config
         embed_dim = config.hidden_size
 
-        if (num_hidden_layers_override is None
-                or num_hidden_layers_override == config.num_hidden_layers):
-            self.need_post_layernorm = True
-        elif num_hidden_layers_override > config.num_hidden_layers:
-            raise ValueError(
-                "num_hidden_layers_override cannot be greater than "
-                "num_hidden_layers")
```

Comment:
We should keep this check no?

Reply:
I've re-added the check but inside the `VisionTransformer` class so the if-else statements stay together.